internal/scorecard/alpha/scorecard.go

```diff
@@ -105,25 +104,30 @@ func (o Scorecard) selectTests() []Test {
 	return selected
 }
 
-// runTest executes a single test
-func (o Scorecard) runTest(ctx context.Context, test Test) (result v1alpha2.ScorecardTestResult, err error) {
+// RunTest executes a single test
+func (r PodTestRunner) RunTest(ctx context.Context, test Test) (result *v1alpha2.ScorecardTestResult, err error) {
 
 	// Create a Pod to run the test
-	podDef := getPodDefinition(test, o)
-	pod, err := o.Client.CoreV1().Pods(o.Namespace).Create(ctx, podDef, metav1.CreateOptions{})
+	podDef := getPodDefinition(test, r.TestConfig)
+	pod, err := r.TestConfig.Client.CoreV1().Pods(r.TestConfig.Namespace).Create(ctx, podDef, metav1.CreateOptions{})
 	if err != nil {
 		return result, err
 	}
 
-	err = o.waitForTestToComplete(ctx, pod)
+	err = r.TestConfig.waitForTestToComplete(ctx, pod)
```

Comment:
Make this a method on `PodTestRunner`?
```suggestion
	err = r.waitForTestToComplete(ctx, pod)
```

Reply:
made it a method on PodTestRunner as you suggested.